### DISCUSSION TITLE
Why do PRs fail the syntax step?

### DISCUSSION THREAD

**Discussion by cesarcoatl** [Homebrew/discussions#442] [Casks] [answered]
This is the second time this has occurred to me, and I've seen some PRs assigend the `ci-syntax-only` label.

What is the reason this keeps occurring?

Failed PR: [Homebrew/cask-versions#10153](https://github.com/Homebrew/homebrew-cask-versions/pull/10153)
Successful PR: [Homebrew/cask-versions#10156](https://github.com/Homebrew/homebrew-cask-versions/pull/10156)

Another failed PR: [Homebrew/cask-versions#10164](https://github.com/Homebrew/homebrew-cask-versions/pull/10164)

**Comment by SMillerDev** [comment]
Because `brew style` failed to run on the repository as it is in your pr. It could be pre-existing or you broke it.

Ci-syntax-only is to only run this test (if nothing changed that affects the install)

**Reply by cesarcoatl** [reply]
10153 and 10156 were exactly the same, yet one failed and the other didn't.

`brew bump-cask-pr` performs a style check, and they've all passed.

Nothing has been broken on my end.

**Reply by cesarcoatl** [reply]
See PRs: 10162, 10163 and 10164.

**Reply by SMillerDev** [reply]
Again, it could be pre-existing. If the style checks succeeded locally that seems to be the situation.

**Reply by cesarcoatl** [reply]
All PRs were "simple version bumps", and I can assure you that this failed syntax was not occurring a week ago.

**Reply by carlocab** [reply]
It could also just be CI being temperamental. GitHub has been doing maintenance on their macOS runners, so that's lead to some issues with them.

**Reply by cesarcoatl** [reply]
Thanks, @carlocab. This is by far a better response than any I've gotten so far.

After 10164 failed the first time I manually triggered a review, and it passed the second time without changing anything.

So the "pre-existing" and the "you broke it" syntax issues don't seem to be the root cause.

**Reply by SMillerDev** [reply]
Someone could have made a commit to either the main homebrew/brew or cask branch that fixed or disabled the style check that was failing. 

The most likely scenario for style checks failing is "the style is deemed wrong". It might not be your fault, but it's more likely to me than "Githubs action runners somehow made a single check fail".

**Reply by carlocab** [reply]
It might be, but looking at the CI run: https://github.com/Homebrew/homebrew-cask-versions/pull/10153/checks?check_run_id=1629365677

The CI run didn't even start, so an actual audit failure seems unlikely. I've seen this already happen a few times (cf. Rylan's message about CI failures on Slack), and it looks more like CI randomly failing rather than an actual style problem.

**Reply by cesarcoatl** [reply]
@SMillerDev, let's take a look at `azure-data-studio-insiders`.

Here is some historical reference regarding all PRs:
[10104](https://github.com/Homebrew/homebrew-cask-versions/pull/10104): passed
[10112](https://github.com/Homebrew/homebrew-cask-versions/pull/10112): passed
[10113](https://github.com/Homebrew/homebrew-cask-versions/pull/10113): passed
[10125](https://github.com/Homebrew/homebrew-cask-versions/pull/10125): passed
[10133](https://github.com/Homebrew/homebrew-cask-versions/pull/10133): passed
[10136](https://github.com/Homebrew/homebrew-cask-versions/pull/10136): passed
[10146](https://github.com/Homebrew/homebrew-cask-versions/pull/10146): passed
[10151](https://github.com/Homebrew/homebrew-cask-versions/pull/10151): passed
[10153](https://github.com/Homebrew/homebrew-cask-versions/pull/10153): failed
[10157](https://github.com/Homebrew/homebrew-cask-versions/pull/10157): passed

Maybe this can eliminate the "style" issue, and convince you about a CI issue.

**Comment by cesarcoatl** [comment]
Incoming PR [Homebrew/homebrew-cask-versions#10166](https://github.com/Homebrew/homebrew-cask-versions/pull/10166).

Terminal output:
```bash
$ brew bump-cask-pr --version 1.26.0,bc9719d5a8679f35dc0dbb4b27cf3437b27f50cb --sha256 b5f5cfa305f3656b3da2a42fea994eed3811c399b68baf0c77c8105ee78d1613 azure-data-studio-insiders
Updating Homebrew...
==> Auto-updated Homebrew!
Updated Homebrew from d820c9a91 to b678af3fa.
Updated 6 taps (homebrew/cask-versions, homebrew/core, homebrew/cask, homebrew/bundle, homebrew/cask-fonts and homebrew/cask-drivers).
==> New Formulae
yq@3
==> Updated Formulae
Updated 15 formulae.
==> Renamed Formulae
gtk+4 -> gtk4
==> New Casks
stringz
==> Updated Casks
Updated 9 casks.

==> replace /version\s+["']1\.26\.0,30054e632c75f8df0ea351f2c51b8ec7d5fa5be5["']/m with "version \"1.26.0,bc9719d5a8679f
==> replace /sha256\s+["']79283632d92758c2737622b4d2d8b6d9ffc52eab4d5b8f32c775bea6e72800fc["']/m with "sha256 \"b5f5cfa3
audit for azure-data-studio-insiders: passed

1 file inspected, no offenses detected
M	Casks/azure-data-studio-insiders.rb
Switched to a new branch 'bump-azure-data-studio-insiders-1.26.0-bc9719d5a8679f35dc0dbb4b27cf3437b27f50cb'
[bump-azure-data-studio-insiders-1.26.0-bc9719d5a8679f35dc0dbb4b27cf3437b27f50cb f4428e6e1] Update azure-data-studio-insiders from 1.26.0,30054e632c75f8df0ea351f2c51b8ec7d5fa5be5 to 1.26.0,bc9719d5a8679f35dc0dbb4b27cf3437b27f50cb
 1 file changed, 2 insertions(+), 2 deletions(-)
Enumerating objects: 7, done.
Counting objects: 100% (7/7), done.
Delta compression using up to 4 threads
Compressing objects: 100% (4/4), done.
Writing objects: 100% (4/4), 550 bytes | 550.00 KiB/s, done.
Total 4 (delta 3), reused 0 (delta 0), pack-reused 0
remote: Resolving deltas: 100% (3/3), completed with 3 local objects.
remote: 
remote: Create a pull request for 'bump-azure-data-studio-insiders-1.26.0-bc9719d5a8679f35dc0dbb4b27cf3437b27f50cb' on GitHub by visiting:
remote:      https://github.com/thecesrom/homebrew-cask-versions/pull/new/bump-azure-data-studio-insiders-1.26.0-bc9719d5a8679f35dc0dbb4b27cf3437b27f50cb
remote: 
To https://github.com/thecesrom/homebrew-cask-versions.git
 * [new branch]          bump-azure-data-studio-insiders-1.26.0-bc9719d5a8679f35dc0dbb4b27cf3437b27f50cb -> bump-azure-data-studio-insiders-1.26.0-bc9719d5a8679f35dc0dbb4b27cf3437b27f50cb
Branch 'bump-azure-data-studio-insiders-1.26.0-bc9719d5a8679f35dc0dbb4b27cf3437b27f50cb' set up to track remote branch 'bump-azure-data-studio-insiders-1.26.0-bc9719d5a8679f35dc0dbb4b27cf3437b27f50cb' from 'https://74f1fc95a7f30f492c2a754c690f35fd56d8b892@github.com/thecesrom/homebrew-cask-versions.git'.
```

**Reply by carlocab** [reply]
Don't sweat it too much; it's not just you. Seems to have happened to this CI run as well: https://github.com/Homebrew/homebrew-cask-versions/pull/10165/checks?check_run_id=1651464066

**Reply by cesarcoatl** [reply]
Hopefully a maintainer might be able to scale this issue.

**Comment by cesarcoatl** [comment, marked as answer]
Answered by @jonchang [here](https://github.com/Homebrew/discussions/discussions/385#discussioncomment-264008)

> Folks, this is a Github issue. There is not a whole lot we can do until their support gets back to us.
